java/api/src/main/java/org/ray/api/Ray.java

```diff
@@ -117,6 +118,26 @@ public static void shutdown() {
     return runtime.wait(waitList, waitList.size(), Integer.MAX_VALUE);
   }
 
+  /**
+   * Wrap a {@link Runnable} with necessary context capture.
```

Comment:
Document that if users want to use Ray API in their own threads, they should wrap their `Runnable` objects with this method.